README.md

```diff
@@ -38,7 +38,6 @@ sub-crates that can be used independently or together.
 | [ibc-testkit](ibc-testkit)           | Provides testing toolkit to aid `ibc-rs` and host chains in writing integration tests.               |
 | [ibc-query](ibc-query)               | Contains utility traits and implementations for querying states of an integrated IBC module.         |
 | [ibc-derive](ibc-derive)             | Derive macros for `ClientState` and `ConsensusState` traits, reducing boilerplate.                   |
-| [cosmwasm](cosmwasm)                 | Contains libraries that implement `ibc-rs` modules as CosmWasm contracts.                            |
```

Comment:
Can we add this project now under the `ibc-rs in the wild` section?

Reply:
[32608ee](https://github.com/cosmos/ibc-rs/pull/1315/commits/32608eef115ba378a160a7c40c0549510fd450ca)